dronegen/push.go

```diff
@@ -164,16 +164,13 @@ func pushPipeline(b buildType) pipeline {
 func sendErrorToSlackStep() step {
 	return step{
 		Name:  "Send Slack notification",
-		Image: "plugins/slack",
+		Image: "plugins/slack:1.4.1",
 		Settings: map[string]value{
 			"webhook": {fromSecret: "SLACK_WEBHOOK_DEV_TELEPORT"},
 			"template": {raw: `*{{#success build.status}}✔{{ else }}✘{{/success}} {{ uppercasefirst build.status }}: Build #{{ build.number }}* (type: ` + "`{{ build.event }}`" + `)
```

Comment:
I would put `{{ build.event }}` and `${DRONE_STAGE_NAME` together:

    "pipeline: `{{ build.event }}` / `${DRONE_STAGE_NAME}`"

and get rid of the text "artifact build failed" - we know it failed, it's why we're seeing the message (and the red, the cross and the word "Failure" kind of give it away too :-) )

All this would reduce it to two lines - more than enough and has all the same info